dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/workflow/WorkflowResourceIntegrationTest.java

```diff
@@ -1,23 +1,92 @@
 package com.dotcms.rest.api.v1.workflow;
```

Comment:
Should this class be in here?

Reply:
true It shouldn't have been modified it was me getting code example from multipart request
My IDE added all the imports and I guess I added a couple of lines